docs/index.md

```diff
@@ -31,6 +31,12 @@ sbt new typelevel/typelevel.g8
 This will guide you through the basic setup to create a new project with **sbt-typelevel** and **sbt-typelevel-site**.
 Check out the [typelevel.g8](https://github.com/typelevel/typelevel.g8) project for more details.
 
+Alternatively, if you don't want to use Giter8, you can add a file called: `.scalafmt.conf` to your project root directory with the contents:
```

Comment:
I suggest we move this to the FAQ. Unfortunately adding a `.scalafmt.conf` by itself is not the only thing you need to do to completely avoid the giter 8, so I think this might be confusing :)